File: pythua.py
def send(text):
  f = open("file.txt","w")
  f.write(text)
  f.close()
  return

def read(erase):
  f = open("file.txt")
  text = f.read()
  f.close()
  if erase == True:
    f.open("file.txt","w")
    f.write("")
    f.close()
  return text

def wait_until_signal(message):
  while True:
    if read() == message:
      break
  read(True)
  

File: pythua.lua
function readfile(erase)
  file = io.open("file.txt", "r")
  io.input(file)
  text = io.read()
  io.close()
  if erase == true:
    file = io.open("file.txt", "w+")
    io.output(file)
    io.write("")
    io.close()
  return text
end

function writefile(text)
  file = io.open("file.txt", "w+")
  io.output(file)
  io.write(text)
  io.close()
end

function wait_until_signal(text)
  while true do
    if readfile(false) == text
      break
    end
  end
  readfile(true)
end
